src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/MyExtensionsApp.Shared.projitems

```diff
@@ -152,11 +152,31 @@
     <Content Include="$(MSBuildThisFileDirectory)appsettings.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\back.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\help_outline.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\close.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\notifications.png" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\share.png" />
+    <Content Include="$(MSBuildThisFileDirectory)Assets\Icons\back.scale-100.png" />
```

Comment:
should we use a wildcard here?

Reply:
It's a copy of the commerce app, and a shared project which is modified by VS, so it won't stay a wildcard :)